programs/sndfile-convert.c

```diff
@@ -114,6 +114,13 @@ report_format_error_exit (const char * argv0, SF_INFO * sfinfo)
 		exit (1) ;
 		} ;
 
+	int channels = sfinfo->channels;
```

Comment:
This fails in travis. See https://travis-ci.org/erikd/libsndfile/jobs/335882223

That variable declaration will need to go at the top of the block.